share/spice/brainy_quote/brainy_quote.css

```diff
@@ -1,34 +1,46 @@
-.brainy__quote--container {
+.zci--brainy_quote .container {
     overflow: hidden;
-    margin-left: -2.8em;
+    margin-left: 0.02em;
+    font-size: 1.5em;
+    font-weight: 300;
+
 }
 
-.brainy__quote {
-    quotes: "“" "”";
-    font-size: 1.5em;
+.zci--brainy_quote .left__quote
+{
+    color: #d0d0d0;
```

Comment:
Let's use our new text-color classes to apply the colors here: 
https://duckduckgo.com/styleguide#txt-n-color
In this case, `tx-clr--platinum-dark`